cosmwasm/contracts/global-accountant/tests/modify_balance.rs

```diff
@@ -18,48 +16,62 @@ fn simple_modify() {
         amount: Uint256::from(300u128),
         reason: "test".into(),
     };
-    let modification = to_binary(&m).unwrap();
-
-    let signatures = wh.sign(&modification);
-    let resp = contract
-        .modify_balance(modification, index, signatures)
-        .unwrap();
 
-    let evt = Event::new("wasm-Modification")
-        .add_attribute("sequence", serde_json_wasm::to_string(&m.sequence).unwrap())
-        .add_attribute("chain_id", serde_json_wasm::to_string(&m.chain_id).unwrap())
-        .add_attribute(
-            "token_chain",
-            serde_json_wasm::to_string(&m.token_chain).unwrap(),
-        )
-        .add_attribute(
-            "token_address",
-            serde_json_wasm::to_string(&m.token_address).unwrap(),
-        )
-        .add_attribute("kind", serde_json_wasm::to_string(&m.kind).unwrap())
-        .add_attribute("amount", serde_json_wasm::to_string(&m.amount).unwrap())
-        .add_attribute("reason", serde_json_wasm::to_string(&m.reason).unwrap());
-
-    resp.assert_event(&evt);
-
-    let actual = contract.query_modification(m.sequence).unwrap();
-    assert_eq!(m, actual);
+    // Test modification via both instantiate + migrate channels.
+    let instantiate_msgs = vec![
```

Comment:
minor nit: this could also be an array instead of a vec (just remove the `vec!` prefix).